docs/pages/loaders/index.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        Loading indicators are used to convey to the user that they will need to wait some time for a software- or hardware-related operation to complete. They can be used to convey a range of ongoing processes, including network, disk and CPU operations.
+      </p>
+      <p>
+        Generally, loading indicators should be used if the wait period is longer than a second.
+      </p>
+    </DocsPageSection>
+
+    <DocsPageSection title="Using linear loaders" anchor="#linearloaders">
+      <p>
+        Use linear loaders for the following:
+      </p>
+      <ul>
+        <li>Loading initial page content</li>
+        <li>Representing a progress of a measurable task</li>
```

Comment:
"Representing *a* progress..." sounds off to me.

Perhaps it should be:

"Representing _the_ progress..." or just "Representing progress"